Mathlib/Data/Fintype/BigOperators.lean

```diff
@@ -281,6 +281,11 @@ open Finset
 
 variable {α₁ : Type*} {α₂ : Type*} {M : Type*} [Fintype α₁] [Fintype α₂] [CommMonoid M]
 
+@[to_additive]
+theorem prod_sum_univ [Fintype α] [Fintype γ] (f : α ⊕ γ → M) :
+    ∏ x, f x = (∏ x, f (Sum.inl x)) * ∏ x, f (Sum.inr x) := by
+  rw [← univ_disjSum_univ, prod_disj_sum]
+
```

Comment:
This is identical to `Fintype.prod_sum_type` a few lines below, isn't it?

Reply:
Oops!